R-package/tests/testthat/test_lgb.unloader.R

```diff
@@ -2,7 +2,13 @@ VERBOSITY <- as.integer(
     Sys.getenv("LIGHTGBM_TEST_VERBOSITY", "-1")
 )
 
+CALCULATING_TEST_COVERAGE <- Sys.getenv("R_COVR") == "true"
```

Comment:
Since we end up using non-public environmental variable, I believe it would be safer to check that variable is just set to something. This will allow us to not track internals of `covr` package and adjust our code if/when they change this piece of their code
https://github.com/r-lib/covr/blob/b86f3c3640fcf415ace12b5e3aaf834cd4c5d3bd/R/covr.R#L164-L165

```suggestion
CALCULATING_TEST_COVERAGE <- Sys.getenv("R_COVR", unset = "unset") != "unset"
```

Reply:
works for me!